How to use go-bindata in OpenBitmap?

I use https://github.com/jteeuwen/go-bindata turn image to Go code.
Then how to use https://github.com/go-vgo/robotgo `OpenBitmap`?

```
----- Test Dir
 |-- asset/asset.go
 |-- main.go
 |-- test.png

$  go-bindata -o=asset/asset.go -pkg=asset ./test.png        // generates Go code from png
$  go run main.go

panic: interface conversion: interface {} is []uint8, not string

goroutine 1 [running]:
github.com/go-vgo/robotgo.OpenBitmap(0xc042067f18, 0x1, 0x1, 0x0)
        C:/Users/Administrator/go/src/github.com/go-vgo/robotgo/robotgo.go:766 +0x1a6
main.main()
        C:/Users/Administrator/go/src/test/main.go:14 +0xcd
exit status 2
```

main.go
```go
package main

import (
	"asset"
	"fmt"
	"reflect"

	"github.com/go-vgo/robotgo"
)

func main() {

	data, _ := asset.Asset("test.png")
	bitmap := robotgo.OpenBitmap(data)
	fmt.Println("...bitmap", bitmap, reflect.TypeOf(bitmap))

}

```

--- a/robotgo.go
+++ b/robotgo.go
@@ -848,12 +848,12 @@ func SaveBitmap(args ...interface{}) string {
 // }
 
 // TostringBitmap tostring bitmap to C.char
-func TostringBitmap(bit C.MMBitmapRef) *C.char {
+func TostringBitmap(bit C.MMBitmapRef) string {
 	// str_bit := C.aTostringBitmap(bit)
 	strBit := C.tostring_Bitmap(bit)
 	// fmt.Println("...", str_bit)
 	// return str_bit
-	return strBit
+	return C.GoString(strBit)
 }
 
 // ToBitmap trans C.MMBitmapRef to Bitmap

